### DISCUSSION TITLE
Free Unicode symbols

### DISCUSSION THREAD

**Discussion by Crissov** [FortAwesome/Font-Awesome#20768] [Feature Requests]
Please adopt a policy to add all symbols that have a Unicode counterpart as part of the Free set. #222

**Comment by Legitti** [comment]
+1

**Comment by tagliala** [comment]
Hi

> add all symbols that have a Unicode counterpart as part of the Free set

How many icons are we talking about?

**Comment by Legitti** [comment]
> Hi
> 
> > add all symbols that have a Unicode counterpart as part of the Free set
> 
> How many icons are we talking about?

At least all the emojis would be so cool. And maybe special characters that his prob talking about.

**Comment by tagliala** [comment]
Could you please provide a number? I think I cannot leave this open in the free repository

https://github.com/FortAwesome/Font-Awesome/issues/new?title=Icon%20request:%20icon-name&template=icon-request.md

> - [ ] This is a single icon or matched pair (Ex: `lock` / `unlock`)

**Comment by Crissov** [comment]
I see how this could have been misunderstood. I do not propose that Font Awesome adds all symbols that are available in Unicode, which would be thousands probably. (Symbola is a free font with this stated goal.) Iʼm just asking Fort Awesome to not make any new FA symbols only available in Pro if they clearly have a corresponding Unicode character.

It would be an objective criterion in an otherwise seemingly arbitrary decision process. I believe only a handful of existing Pro symbols would need to be reassessed.

This idea was spawned by the discussion in #866, although the symbol in question there does not have a Unicode equivalent.  🎃 requested in #10055, however, would be a symbol that such a rule would affect.
